acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -0,0 +1,19 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No arista hosts present" unless platforms.any? { |val| /^eos-/ =~ val }
+test_name 'Arista Switch Pre-suite' do
+  masters = select_hosts({:roles => ['master', 'compile_master']})
+  switchs = select_hosts({:platform => ['eos-4-i386']})
+
+  step 'install Arista Module on masters' do
+    masters.each do |node|
+      on(node, puppet('module','install','aristanetworks-netdev_stdlib_eos'))
+    end
+  end
```

Comment:
Similar comment about facter acceptance doesn't have a master, and whether `select_hosts` will return nil or an empty array? If it's empty array, then this seems fine, if not, it will raise when we try to call `nil.each`